arch/RISCV/RISCVDisassembler.c

```diff
@@ -0,0 +1,397 @@
+//===-- RISCVDisassembler.cpp - Disassembler for RISCV --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+/* Capstone Disassembly Engine */
+/* By Rodrigo Cortes Porto <porto703@gmail.com>, 2018 */
+
+#ifdef CAPSTONE_HAS_RISCV
+
+#include <stdio.h>		// DEBUG
+#include <stdlib.h>
+#include <string.h>
+
+#include "../../cs_priv.h"
+#include "../../utils.h"
+
+#include "../../MCInst.h"
+#include "../../MCInstrDesc.h"
+#include "../../MCFixedLenDisassembler.h"
+#include "../../MCRegisterInfo.h"
+#include "../../MCDisassembler.h"
+#include "../../MathExtras.h"
+#include "RISCVBaseInfo.h"
+
+///
+static uint64_t getFeatureBits(int mode) {
```

Comment:
to be consistent with existing coding style, please put the open bracket of the function on the new line, but not at the end of the same line.

(here & elsewhere)